src/test/java/InputParserTest.java

```diff
@@ -20,13 +20,13 @@ public void inputTest() throws IOException {
         String input6 = "deadline will never surface /by 22/11/2039";
         String input7 = "more random nonsense";
 
-        inputParser.actionDeterminer(input1);
-        inputParser.actionDeterminer(input2);
-        inputParser.actionDeterminer(input3);
-        inputParser.actionDeterminer(input4);
-        inputParser.actionDeterminer(input5);
-        inputParser.actionDeterminer(input6);
-        inputParser.actionDeterminer(input7);
+        inputParser.determineAction(input1);
```

Comment:
Nice work! This is a thorough test case and you considered many different cases.